__init__.py

```diff
@@ -76,7 +76,7 @@
 # Define Version
 
 majorVersion = 6
-minorVersion = 13
+minorVersion = 15
```

Comment:
Fixed also in https://github.com/DIRACGrid/DIRAC/pull/2712 (commit: https://github.com/fstagni/DIRAC/commit/0a82caa2d8cbb386bdb4fc4a1f23751591fc6ca8)